src/rdkafka_conf.c

```diff
@@ -829,6 +829,22 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "Path to client's keystore (PKCS#12) used for authentication.",
           _UNSUPPORTED_SSL
         },
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
```

Comment:
Needs to use _UNSUPPORTED_SSL_1_1 macro, see above.

Reply:
@edenhill Can you explain how is going to compile since ssl.engine_location will no longer be available in no-ssl scenarios ?

Reply:
putting the  #if OPENSSL_VERSION_NUMBER >= 0x10100000  back along with _UNSUPPORTED_SSL_1_1